Pipfile

```diff
@@ -6,10 +6,11 @@ verify_ssl = true
 [dev-packages]
 
 [packages]
-thoth-python = "*"
+thoth-python = {git = "https://github.com/bissenbay/python.git",ref = "hash-calc"}
```

Comment:
OK for now. We should release a new version with your changes included otherwise bots will not manage this dependency.

Reply:
have we release a new version? whats the test coverage? ;)

Reply:
Not yet. Yes, I'll try to work on making some tests